x/mongo/driver/driver.go

```diff
@@ -10,7 +10,6 @@ import (
 // Deployment is implemented by types that can select a server from a deployment.
 type Deployment interface {
 	SelectServer(context.Context, description.ServerSelector) (Server, error)
-	SupportsRetryWrites() bool
```

Comment:
We have a number of implementations of the `Deployment` interface, each of which has a `SupportsRetryWrites` function. We should go through and actually remove those functions to make sure we don't have dead code. I'd recommend doing `git grep SupportsRetryWrites() bool` or using vscode's "find all implementations" feature to find these methods and remove them.

Reply:
got rid of the deadcode related to ` supportsRetryWrites`